File: app/src/main/java/framework/sql/SearchSuffixDb.java
package framework.sql;

/**
 * by y on 2016/11/22
 */

public class SearchSuffixDb {
    public static void insert(String suffix) {
        if (GreenDaoDbUtils.isEmpty(suffix)) {
            GreenDaoDbUtils.getSuffixDb().insert(new SearchSuffix(null, suffix));
        }
    }
}

File: app/src/main/java/framework/sql/sqlutil/SearchSuffixDb.kt
package framework.sql.sqlutil


import framework.sql.SearchSuffix

/**
 * by y on 2016/11/22
 */

object SearchSuffixDb {
    fun insert(suffix: String) {
        if (GreenDaoDbUtils.isEmpty(suffix)) {
            GreenDaoDbUtils.suffixDb.insert(SearchSuffix(null, suffix))
        }
    }
}
